theme/mixins/backdrop.less

```diff
@@ -7,4 +7,5 @@
   background-color: @sk-black;
   opacity: 0.7;
   transition: opacity 0.3s;
+  z-index: @z-index-negative;
```

Comment:
After this PR it's not breaking current behaviour but everyone else is discouraged to opt-in to backdrop until work on it is fully-finished